xgboost/xgboost_cv_terminatorcallback_integration.py

```diff
@@ -0,0 +1,63 @@
+"""
+Optuna example that demonstrates a pruner for XGBoost.cv.
+
+In this example, we optimize the validation auc of cancer detection using XGBoost.
+We optimize both the choice of booster model and their hyperparameters. Throughout
+training of models, a pruner observes intermediate results and stops unpromising trials,
+and a terminator observes studies and stops optimizations with insignificant changes.
+
+You can run this example as follows:
+    $ python xgboost_cv_integration.py
```

Comment:
Maybe this is a controversial topic, the current file name `xgboost_cv_terminatorcallback_integration.py` is a bit complicated. How about renaming the file?
```suggestion
    $ python xgboost_cv_terminator.py
```